docs/topics/jvm/java-to-kotlin-interop.md

```diff
@@ -338,7 +338,7 @@ public class BB8 implements Robot {
 
 If there are clients that use your Kotlin interfaces compiled without the `-Xjvm-default=all` option, then they may
 be binary-incompatible with the code compiled with this option. To avoid breaking the compatibility with such clients, 
-use the `-Xjvm-default=all` mode and mark interfaces with the `@JvmDefaultWithCompatibility` annotation. 
+use the `-Xjvm-default=all` mode and mark interfaces with the [`@JvmDefaultWithCompatibility` annotation](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default-with-compatibility/). 
```

Comment:
```suggestion
use the `-Xjvm-default=all` mode and mark interfaces with the [`@JvmDefaultWithCompatibility`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default-with-compatibility/) annotation. 
```

Reply:
Don't you think including the word annotation into the link would be better from the SEO perspective?

Reply:
It is pretty similar to the case, when we provide links to particular functions. 
There is no reason to think about SEO: there is no competitive content, annotations with the same name on other sites, etc.

Actually, if users will search for this annotation, the most relevant page for him is this: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default-with-compatibility/